CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 # Changelog
 
+## [v4.3.0] - 2021-03-10
+
+### Added:
+
+- **[dnsprovider]** Add DNS provider for Njalla
+- **[dnsprovider]** Add DNS provider for Domeneshop
+- **[dnsprovider]** Add DNS provider for Hurricane Electric
+- **[dnsprovider]** designate: support for Openstack Application Credentials
+- **[dnsprovider]** edgedns: support for .edgerc file
+
+### Changed:
```

Comment:
It should be noted that MD5 is not supported any more as valid algorithm RFC2136. Maybe something akin to:

```markdown
- **[dnsprovider]** rfc2163: Removed support for MD5 algorithm. The default algorithm is now SHA1.
```

Technically, this is a breaking change, isn't it?

Reply:
> Technically, this is a breaking change, isn't it?

Btw. I'm fine with this being just a minor release, feel free to ignore this remark.